apps-rendering/src/components/Metadata/LiveBlogMetadata.tsx

```diff
@@ -0,0 +1,185 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import { background } from '@guardian/common-rendering/src/editorialPalette';
+import type { ArticleFormat } from '@guardian/libs';
+import { ArticleDesign } from '@guardian/libs';
+import {
+	breakpoints,
+	from,
+	neutral,
+	remSpace,
+	until,
+} from '@guardian/source-foundations';
+import { ToggleSwitch } from '@guardian/source-react-components-development-kitchen';
+import Avatar from 'components/Avatar';
+import Byline from 'components/Byline';
+import CommentCount from 'components/CommentCount';
+import Dateline from 'components/Dateline';
+import Follow from 'components/Follow';
+import LiveblogMetadataLines from 'components/LiveblogMetadataLines';
+import Logo from 'components/Logo';
+import { getFormat } from 'item';
+import type { Item } from 'item';
+import { useState } from 'react';
+import type { FC } from 'react';
+import { darkModeCss } from 'styles';
+import { defaultStyles, defaultTextStyles } from './Metadata.defaults';
+
+const blogStyles = css`
+	display: block;
+	margin-bottom: 0;
+
+	${from.mobileLandscape} {
+		padding: 0 ${remSpace[5]};
+	}
+
+	${from.tablet} {
+		width: ${breakpoints.tablet}px;
+		margin: 0 auto;
+	}
+
+	${from.desktop} {
+		padding: 0;
+		width: 100%;
+	}
+`;
+
+// This styling function is only temprarily used and will be removed
+// after the liveblog header is completed and background colours
+// are added
+const tempraryBackgroundStyle = (format: ArticleFormat): SerializedStyles => {
```

Comment:
It sounds like these may not be temporary any more, so they could probably be merged into the `blogStyles`?

Reply:
Yeah actually that whole thing is complete overkill anyway, the only difference between the two cases was the background colour on mobile.